csharp-linear-search/csharp-linear-search.cs

```diff
@@ -18,15 +18,15 @@ public static void Main()
                 a[i] = Int32.Parse(s1);
             }
             
-            Console.WriteLine("Enter Search element\n");
+            Console.WriteLine("Enter Search element \n");
             string s3 = Console.ReadLine();
             int x2 = Int32.Parse(s3);
             for (int i = 0; i < x; i++)
             {
                 if (a[i] == x2)
                 {
                    
-                    Console.WriteLine("Search successful");
+                    Console.WriteLine("Search successful \n");
```

Comment:
What is the significance of this change? [Console.WriteLine](https://docs.microsoft.com/en-us/dotnet/api/system.console.writeline) already appends a line break.

Likewise, the earlier change in this PR is simply adding extra whitespace _before_ an existing line break.